modules/latex/latex.js

```diff
@@ -0,0 +1,19 @@
+( function( $ ) {
+	'use strict';
+	$( document.body ).on( 'post-load', function() {
+		// New posts have been added to the page. Re-typeset MathJax if necessary.
+		if ( typeof MathJax !== 'undefined' ) {
+			$( '.infinite-wrap' ).not( '.mathjax-typeset' ).each( function() {
+				var wrap = this;
+
+				MathJax.Hub.Queue( [
+					'Typeset',
+					MathJax.Hub,
+					wrap
+				] );
+
+				$( wrap ).addClass( 'mathjax-typeset' );
+			} );
```

Comment:
One thing I haven't tried yet is queuing a `Typeset` action from within `infinity.js` itself. The `post-load` event may not be happening at the absolute best time for it and could therefore be done better elsewhere. I know that's not a good solution, but I am curious.